src/main/scala/mesosphere/marathon/api/akkahttp/v2/GroupsController.scala

```diff
@@ -102,7 +103,26 @@ class GroupsController(
 
   def updateGroup(groupId: PathId): Route = ???
 
-  def deleteGroup(groupId: PathId): Route = ???
+  def deleteGroup(groupId: PathId)(implicit identity: Identity): Route = {
+    forceParameter { force =>
+      val version = Timestamp.now()
+
+      def deleteGroupEither(rootGroup: RootGroup) = {
```

Comment:
Hm, I know we had this pattern before but this seems odd in comparison to

```scala
def deleteGroup(rootGroup: RootGroup): Either[Rejection, DeploymentPlan]
```

why limit the compiler here?